codegen/collector/executor.go

```diff
@@ -104,17 +108,16 @@ func (o *OpenApiProtocExecutor) Execute(protoFile string, toFile string, imports
 	_, fileName := filepath.Split(protoFile)
 	directoryName := fileName[0 : len(fileName)-len(filepath.Ext(fileName))]
 
+	baseArgument := "--openapi_out=yaml=true,single_file=false,include_description=true,multiline_description=true,proto_oneof=true,int_native=true"
+	baseArgument = fmt.Sprintf("%s,enum_as_int_or_string=%v", baseArgument, o.EnumAsIntOrString)
+	baseArgument = fmt.Sprintf("%s,additional_empty_schema=%v", baseArgument, strings.Join(o.MessagesWithEmptySchema, "+"))
+	baseArgument = fmt.Sprintf("%s,disable_kube_markers=%v", baseArgument, o.DisableKubeMarkers)
+	baseArgument = fmt.Sprintf("%s,ignored_kube_marker_substrings=%v", baseArgument, strings.Join(o.IgnoredKubeMarkerSubstrings, "+"))
```

Comment:
If you want to make this more readable, I'd group these options into a slice and use `strings.Join()`.

Reply:
Yeah honestly I should have done that. I think this is better than what we have, so if we revisit this and add another argument we could improve this again.